File: edudoc/src/app/model/interfaces/custom/merge.dto.ts
import { ICaseLoadCptCode } from '../case-load-cpt-code';
import { ICaseLoadGoal } from '../case-load-goal';
import { ISchoolDistrictRoster } from '../school-district-roster';
import { IStudent } from '../student';
import { IStudentTherapy } from '../student-therapy';

export interface IMergeDTO {
    Roster: ISchoolDistrictRoster;
    Student: IStudent;
    StudentIds: number[];
    ParentalConsentTypeId?: number;
    ParentalConsentEffectiveDate?: Date;
    MergingIntoStudentId?: number;
    StudentMergeInto?: IStudent;
}

File: edudoc/src/Model/DTOs/MergeDto.cs
using System.Data.SqlClient;
using System.Data.SqlClient;
using System;
using System.Collections.Generic;

namespace Model.DTOs
{
    public class MergeDTO
    {
        public SchoolDistrictRoster Roster
        {
            get;
            set;
        }

        public Student Student
        {
            get;
            set;
        }

        public Student StudentMergeInto
        {
            get;
            set;
        }

        public int? ParentalConsentTypeID
        {
            get;
            set;
        }

        public int? MergingIntoStudentId
        {
            get;
            set;
        }

        public DateTime? ParentalConsentEffectiveDate
        {
            get;
            set;
        }

        public List<int> StudentIds { get; set; }

    }
}
